toast-tk-rest-utils/src/main/java/io/toast/tk/core/rest/InspectPage.java

```diff
@@ -9,18 +9,32 @@
 public class InspectPage {
 
 	public String name;
+	
+	public Double erreur;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make erreur a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)